.github/workflows/scripts/linux/generate-cmake.sh

```diff
@@ -3,8 +3,8 @@
 set -e
 
 if [ "${COMPILER}" = "gcc" ]; then
-  export CC=gcc
-  export CXX=g++
+  export CC=gcc-8
+  export CXX=g++-8
```

Comment:
Why not use gcc 9 or 10? https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#language-and-runtime

Reply:
We're on 18.04
IIRC it made the appimage easier

Reply:
18.04 also has gcc 9 and 10: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md#language-and-runtime